library/common/main_interface.h

```diff
@@ -163,9 +163,11 @@ envoy_status_t register_platform_api(const char* name, void* api);
  * Initialize an engine for handling network streams.
  * @param callbacks, the callbacks that will run the engine callbacks.
  * @param logger, optional callbacks to handle logging.
+ * @param event_tracker, optional event tracker for the emission of events.
```

Comment:
Is this optional?

Reply:
(Asking because you're explicitly registering a null tracker in the cc path.)

Reply:
nah, it's not - used to be initially since it was a pointer but I changed it to be a struct so this no longer holds. Updating.